### PR TITLE
Add uic command arguments

### PR DESCRIPTION
**Pull request by k831-kim**
uic commands (dme get/set, dme peer get/set) have two more arguments. (attr_set_type, gen_selectoe_index)
so, i raise pr for that arguments.

### CODE REVIEW

**Comment by minwooim** [comment]
@arthur-simchaev-wdc ,

Could you please review this one?

**Comment by arthur-simchaev-sndk** [comment]
Hi , the gen selector index was added as the part of Eye Monitor commit.
Can you check if it cover your request?

**Comment by k831-kim** [comment]
Hi, I checked that commit.  it can cover my gen select commit. i will remove gen select commit. 
but attr set type commit is not covered. could you review that?

**Comment by arthur-simchaev-sndk** [comment]
Yes, sure

Regards
Arthur

File: options.c [hunk 1]
```diff
@@ -37,6 +37,7 @@ static int verify_region(struct tool_options *options);
 static int verify_and_set_hmr_method(struct tool_options *options);
 static int verify_and_set_hmr_unit(struct tool_options *options);
 static int verify_sg_struct(struct tool_options *options);
+static int verify_and_set_attrsettype(struct tool_options *options);
```
Comment (arthur-simchaev-sndk) [inline options.c]: I think the function is redundant. You need add  he new parameter to the long_opion : 
 static struct option long_opts[] = {
		{"peer", no_argument, NULL, 'u'}, /* UFS device */
		{"local", no_argument, NULL, 'l'}, /* UFS host*/
                {"static", no_argument, opt->set_type, 1},
2) set_type should be init to 0 in initialized_options function
3) Update the uic help accordingly

**Review by arthur-simchaev-sndk — COMMENTED** [review]
Done, see the comments

File: options.c (hunk 1 above)
Comment (k831-kim) [reply options.c]: okay i will update that.

File: options.c [hunk 2]
```diff
@@ -52,6 +52,7 @@ int init_options(int opt_cnt, char *opt_arr[], struct tool_options *options)
 	static struct option long_opts[] = {
 		{"peer", no_argument, NULL, 'u'}, /* UFS device */
 		{"local", no_argument, NULL, 'l'}, /* UFS host*/
+		{"static", no_argument, NULL, 's'},
```
Comment (k831-kim) [inline options.c]: I try to alloc below code
{"static", no_argument, options->set_type, 1},
but, it makes "initializer element is not constant" error
so, i make simple option

**Comment by arthur-simchaev-sndk** [comment]
OK, but "s" option already used by gen selector

**Comment by k831-kim** [comment]
Sorry, i did not check that.
How about make a "S" option for setting the attr_set_type?

**Comment by arthur-simchaev-sndk** [comment]
Yes , you can use it

Regards
Arthur
________________________________

**Comment by k831-kim** [comment]
I updated that. thanks

File: options.c [hunk 3]
```diff
@@ -52,12 +52,13 @@ int init_options(int opt_cnt, char *opt_arr[], struct tool_options *options)
...
 		{"peer", no_argument, NULL, 'u'}, /* UFS device */
 		{"local", no_argument, NULL, 'l'}, /* UFS host*/
+		{"static", no_argument, NULL, 'S'},
 		/* output file for the descriptor file store */
 		{"output_file", required_argument, NULL, 'D'},
 		{"output_mode", required_argument, NULL, 'P'},
 		{NULL, 0, NULL, 0}
 	};
-	static char *short_opts = "t:p:w:i:s:O:L:n:k:m:d:x:y:g:D:P:rocea";
+	static char *short_opts = "t:p:w:i:s:O:L:n:k:m:d:x:y:g:S:D:P:rocea";
```
Comment (arthur-simchaev-sndk) [inline options.c]: not need add S:  option

Comment (k831-kim) [reply options.c]: Thasnks! I fixed it

**Review by arthur-simchaev-sndk — APPROVED** [review]

**Pull request merged**